code/datums/uplink_item.dm

```diff
@@ -376,24 +376,59 @@ var/list/uplink_items = list()
 	gamemodes = list(/datum/game_mode/nuclear)
 	surplus = 40
 
+/datum/uplink_item/dangerous/pdw
+	name = "Gepard PWD"
+	reference = "PWD"
+	desc = "A fully-loaded PWD that fires 7.62x25mm rounds with a 30-round magazine and is compatible with suppressors. Best used on armored targets."
+	item = /obj/item/weapon/gun/projectile/automatic/gepard/syndi
+	cost = 8
+	surplus = 50
+
 /datum/uplink_item/dangerous/carbine
-	name = "M-90gl Carbine"
-	desc = "A fully-loaded three-round burst carbine that uses 30-round 5.56mm magazines with a togglable underslung 40mm grenade launcher."
+	name = "M-90C Carbine"
+	desc = "A fully-loaded three-round burst carbine that uses 30-round 5.56x45mm magazines."
 	reference = "AR"
 	item = /obj/item/weapon/gun/projectile/automatic/m90
 	cost = 18
 	gamemodes = list(/datum/game_mode/nuclear)
 	surplus = 50
 
+/datum/uplink_item/dangerous/carbinecombo
+	name = "M-90gl Carbine"
+	desc = "A fully-loaded three-round burst carbine that uses 30-round 5.56x45mm magazines with a togglable underslung 40mm grenade launcher."
+	reference = "AR"
+	item = /obj/item/weapon/gun/projectile/automatic/m90/gl
+	cost = 23
+	gamemodes = list(/datum/game_mode/nuclear)
+	surplus = 25
+
+/datum/uplink_item/dangerous/grenade
+	name = "CP-40"
+	desc = "A break action 40mm grenade launcher."
+	reference = "GL"
+	item = /obj/item/weapon/gun/projectile/revolver/grenadelauncher/syndi
+	cost = 18
+	gamemodes = list(/datum/game_mode/nuclear)
+	surplus = 10
+
 /datum/uplink_item/dangerous/machinegun
 	name = "L6 Squad Automatic Weapon"
-	desc = "A fully-loaded Aussec Armory belt-fed machine gun. This deadly weapon has a massive 50-round magazine of devastating 7.62x51mm ammunition."
+	desc = "A fully-loaded Aussec Armory belt-fed machine gun. This deadly weapon has a massive 50-round magazine of devastating 5.56x45mm ammunition."
 	reference = "LMG"
 	item = /obj/item/weapon/gun/projectile/automatic/l6_saw
 	cost = 40
 	gamemodes = list(/datum/game_mode/nuclear)
 	surplus = 0
 
+/datum/uplink_item/dangerous/dmr
+	name = "Dragon DMR"
+	desc = "Long range marksman rifle, now you can reach out and kill anyone, anywhere!"
```

Comment:
Not everyone will know what DMR means. Maybe you should expand the acronym in the description?

`"Long range designated marksman rifle, now you can reach out and kill anyone, anywhere!"`

Reply:
Dank Matter Rifle. Screw anti-material rifles, the danker the better.

I honestly don't know what it means without a Google search